crates/paimon/src/spec/types.rs

```diff
@@ -125,8 +330,12 @@ pub struct ArrayType {
 }
 
 impl Display for ArrayType {
```

Comment:
Do types like `ArrayType` and `VarCharType` require an `is_nullable` method?

Reply:
I think the `DataType#is_nullable()` method is enough ?

Reply:
I believe it's not a blocker. We can add this in the future if needed.